content/berks/_index.md

```diff
@@ -50,7 +54,11 @@ creditRole="Capitol Reporter"
 creditEmail="heather@spotlightpa.org"
 creditBio="Interested in learning more? Contact Heather Adams, Berks County Project Lead, at [heather@spotlightpa.org](mailto:heather@spotlightpa.org)"
 >}}
-## A Novel Approach to Reinvigorating Local News
+
+## [Read our editors statement about this study](https://www.spotlightpa.org/news/2023/09/reading-pennsylvania-news-berks-county-spotlight-pennsylvania/)
+
+
+### A Novel Approach to Reinvigorating Local News
```

Comment:
Should this be H2 or H3?

Reply:
Structurally it should be h3, I chose h2 for the aesthetics, but we're going with the link at the top (line 44)